packages/collector/test/tracing/control_flow/pubsub_async_iterator/app.js

```diff
@@ -36,13 +36,13 @@ asyncIterator.pullValue().then(event1 => {
   // Chronologically, everything inside the then-handler this happens after cls.ns.set (see below). Due to custom
   // queueing in pubsub_async_iterator, the cls context would get lost though (unless we fix it).
   log(event1);
-  valuesReadFromCls.push(cls.ns.get('key'));
+  valuesReadFromCls.push(cls.ns.get('key', true));
```

Comment:
@basti1302 , this change is related to your suggestion in the old PR: https://github.com/instana/nodejs/pull/416#discussion_r760925276
There were other changes, so I will tag you in all of them.